site/content/docs/developing/publish-subscribe.en.md

```diff
@@ -70,6 +70,21 @@ subscribe:
 
 Copilot will create a subscription between this worker service's queue and the `ordersTopic` topic from the `api` service. It will also inject the queue URI into the service container under the environment variable `COPILOT_QUEUE_URI`.
 
+The URI of this queue will be injected into the container as an environment variable, `COPILOT_QUEUE_URI`. If you specify
```

Comment:
Looks like first statement here is similar to the one above. Do we need these 2 statements? We can have either of them right?